x-pack/plugins/global_search_providers/server/providers/saved_objects/provider.ts

```diff
@@ -13,6 +13,10 @@ export const createSavedObjectsResultProvider = (): GlobalSearchResultProvider =
   return {
     id: 'savedObjects',
     find: (term, { aborted$, maxResults, preference }, { core }) => {
+      if (!term) {
+        return of([]);
+      }
```

Comment:
Just a quick circuit breaker to avoid unnecessary ES calls until we discuss more in details of the expected behavior for empty term searches in https://github.com/elastic/kibana/issues/78771